src/cart/crt_iv.c

```diff
@@ -364,13 +366,13 @@ crt_ivf_pending_request_add(struct crt_ivns_internal *ivns_internal,
  * invoking fetch completion callback
  */
 static int
-crt_ivf_finalize(struct iv_fetch_cb_info *iv_info, crt_iv_key_t *iv_key,
-		 int output_rc)
+crt_ivf_finalize(struct iv_fetch_cb_info *iv_info,
+		 crt_iv_key_t *iv_key, int output_rc)
```

Comment:
This is a step backwards when it comes to compliance with clang-format, I know not everyone is using the githook yet but this is a white-space only change in the wrong direction.

Reply:
sure, will revert it back.